bind.js

```diff
@@ -24,14 +24,15 @@
 			} else if (Array.isArray(arg)) {
 				classes.push(classNames.apply(this, arg));
 			} else if (argType === 'object') {
-				if (arg.toString === Object.prototype.toString) {
-					for (var key in arg) {
-						if (hasOwn.call(arg, key) && arg[key]) {
-							classes.push(this && this[key] || key);
-						}
-					}
-				} else {
+				if (arg.toString !== Object.prototype.toString && !arg.toString.toString().includes('[native code]')) {
```

Comment:
```suggestion
				if (arg.hasOwnProperty('toString')) {
					classes.push(arg.toString());
					continue;
				}

				if (arg.toString !== Object.prototype.toString && !arg.toString.toString().includes('[native code]')) {
```

We have the option to shortcut `hasOwnProperty` here, but the additional branch hurts typical usage by ~5%. 
I don't think `toString` is typical usage so I have opted out of adding it here.